README.md

```diff
@@ -1,6 +1,6 @@
 # Doctor Mark
 
-An Automatic Page Generation Tool http://jxnblk.com/doctor-mark
+An Automatic Page Generation Tool <http://jxnblk.com/doctor-mark>
```

Comment:
The `<...>` syntax is just a shorthand for when the title and href are the same.
http://daringfireball.net/projects/markdown/syntax#autolink
http://spec.commonmark.org/0.17/#autolinks